lumberjack.go

```diff
@@ -188,13 +193,13 @@ func (l *Logger) rotate() error {
 // openNew opens a new log file for writing, moving any old log file out of the
 // way.  This methods assumes the file has already been closed.
 func (l *Logger) openNew() error {
-	err := os.MkdirAll(l.dir(), 0744)
+	err := os.MkdirAll(l.dir(), l.filePermission())
```

Comment:
I don't think we can assume the same permissions for the directory as for the file.